csa-runner-app/src/main/java/com/farao_community/farao/swe_csa/app/dichotomy/DichotomyRunner.java

```diff
@@ -113,6 +113,14 @@ public RaoResult runDichotomy(CsaRequest csaRequest) throws GlskLimitationExcept
             double ctPtEsMax = expPtEs0 >= 0 ? Math.min(Math.min(-ctRaPtEs.getMinAdmissibleSetpoint(expPtEs0), ctRaEsPt.getMaxAdmissibleSetpoint(expEsPt0)), expPtEs0)
                 : Math.min(Math.min(ctRaPtEs.getMaxAdmissibleSetpoint(expPtEs0), -ctRaEsPt.getMinAdmissibleSetpoint(expEsPt0)), -expPtEs0);
 
+            if (ctFrEsMax != Math.abs(expFrEs0)) {
+                LOGGER.warn("Maximum counter trading FR-ES '{}' is different from initial exchange FR-ES '{}' ", ctFrEsMax, expFrEs0);
+            }
+
+            if (ctPtEsMax != Math.abs(expPtEs0)) {
+                LOGGER.warn("Maximum counter trading PT-ES '{}' is different from initial exchange PT-ES '{}' ", ctPtEsMax, expPtEs0);
+            }
```

Comment:
I think the log is only interesting if max is smaller than exchange
```suggestion
            if (ctFrEsMax < Math.abs(expFrEs0)) {
                LOGGER.warn("Maximum counter trading FR-ES '{}' is smaller than initial exchange FR-ES '{}' ", ctFrEsMax, expFrEs0);
            }

            if (ctPtEsMax < Math.abs(expPtEs0)) {
                LOGGER.warn("Maximum counter trading PT-ES '{}' is smaller than initial exchange PT-ES '{}' ", ctPtEsMax, expPtEs0);
            }
```